net/net_test.go

```diff
@@ -161,6 +167,9 @@ func TestNet_AddThread(t *testing.T) {
 	if len(info3.Logs) != 2 {
 		t.Fatalf("expected 2 logs got %d", len(info3.Logs))
 	}
+	if len(info3.Addrs) == 0 {
+		t.Fatalf("expected more than 0 addresses got %d", len(info3.Addrs))
+	}
```

Comment:
Updated these tests to check for the presence of address information in `thread.Info`